README.md

```diff
@@ -34,11 +35,10 @@ See the [PMM Documentation](https://www.percona.com/doc/percona-monitoring-and-m
 
 Please check our [Documentation](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html) for the actual architecture.
 
-![Overal Architecture](https://docs.percona.com/percona-monitoring-and-management/_images/C_S_Architecture.jpg "Client Server Architecture")
```

Comment:
I think we still need this image here

Reply:
Right, I yanked it by mistake. Put back now.